pkg/provider/gitlab/gitlab.go

```diff
@@ -314,8 +314,9 @@ func (v *Provider) GetTektonDir(_ context.Context, event *info.Event, path strin
 	}
 
 	opt := &gitlab.ListTreeOptions{
-		Path: gitlab.String(path),
-		Ref:  gitlab.String(event.HeadBranch),
+		Path:      gitlab.String(path),
+		Ref:       gitlab.String(event.HeadBranch),
+		Recursive: gitlab.Bool(true),
```

Comment:
yeah I saw this yesterday.. don't know why didn't created a pull request 😝

Reply:
The hard part is the testing 😅